packages/tools/client-debugger/client-debugger-view/src/components/data-object-views/SharedMapView.tsx

```diff
@@ -78,8 +78,8 @@ function getTableValue(data: unknown, _renderChild: RenderChild): React.ReactNod
 		return "null";
 	}
 
-	if (typeof data === ("string" || "number")) {
-		return <>{data}</>;
+	if (typeof data === "string" || typeof data === "number") {
```

Comment:
It occurs to me that this function is basically doing what `renderChild` will do on its own. And since we're no longer immediately prepping for data-editing, it might be worth replacing the use of `getTableValue` with just `renderChild`.

Reply:
I can definitely work on some cleaning tasks to make the logic more concise.